src/cromshell/utilities/io_utils.py

```diff
@@ -0,0 +1,44 @@
+from termcolor import colored
+
+
+def dead_turtle():
+    """Print Dead Turtle"""
+    print(colored("    ,,    ,,     ,,      ", "red"))
```

Comment:
I feel bad mentioning this now, but python's `raw strings` would have saved you from all the escaping.  They force a string to be interpreted as written without any escape characters.  

I feel like we should change the logos into raw strings, but we don't necessarily have to do that now.

Reply:
done